components/automate-ui/src/app/modules/user/user-management/user-management.component.ts

```diff
@@ -88,6 +94,24 @@ export class UserManagementComponent implements OnInit, OnDestroy {
           this.closeCreateModal();
         }
       });
+
+    combineLatest([
+      this.store.select(createStatus),
+      this.store.select(createError)
+    ]).pipe(
+      takeUntil(this.isDestroyed),
+      filter(() => this.createModalVisible),
+      filter(([state, error]) => state === EntityStatus.loadingFailure && !isNil(error)))
```

Comment:
Read an article recently that it should be a best practice to always put `takeUntil` last. If it is not last, then some operators might still leave subscription leaks around. To be clear, the code here does **not** have any issue in that regard (and most of our code would not).

Not suggesting we fix all the non-conforming instances now. Just that going forward it is something to keep in mind for "safe coding". 😁